assets/src/containers/GradeDistribution.js

```diff
@@ -62,51 +62,52 @@ function GradeDistribution (props) {
     const grades = gradeData.map(x => x.current_grade)
 
     const tableRows = [
-      ['Average grade', <strong>{roundToOneDecimal(average(grades))}%</strong>],
-      ['Median grade', <strong>{roundToOneDecimal(median(grades))}%</strong>],
-      ['Number of students', <strong>{gradeData.length}</strong>],
-      showGrade ?
+      ['Average grade', <strong>{gradeData[0].grade_avg}%</strong>],
+      ['Median grade', <strong>{gradeData[0].median_grade}%</strong>],
+      ['Number of students', <strong>{gradeData[0].tot_students}</strong>],
+      !user.admin && showGrade ?
         [
           'My grade',
           <strong>{
             gradeData[0].current_user_grade ?
               `${roundToOneDecimal(gradeData[0].current_user_grade)}%` :
               'There are no grades yet for you in this course'
           }</strong>
-        ] : []
+        ] : [],
     ]
 
     const gradeCheckbox = !user.admin ?
       <> {userSettingLoaded ?
-        <> {'Show my grade'}
+        <> <Typography align='right'>{'Show my grade'}
```

Comment:
This is how I am right aligning the `show my grade` but this takes added space and pushes the histogram little down. Any better way of doing this?

Reply:
Did we decide we didn't like it on the left hand side under the grade statistics?

Reply:
We decided on top right last time, but will find out today if team like the current positioning